server/boot/model-explorer.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function mountLoopBackExplorer(server) {
+  var explorer;
+  try {
+    explorer = require('loopback-explorer');
+    var restApiRoot = server.get('restApiRoot');
+
+    var explorerApp = explorer(server, {basePath: restApiRoot});
+    server.use('/explorer', explorerApp);
```

Comment:
in the pm case... I suspect that it would not itself have any models, so that it would do:

- / -> it's status
- /api -> core API
- /explorer -> core explorer
- /... -> cicada
- * error middleware

oh, but I see your point, explorer might need to be told, at creation time, where it will be mounted, and where the REST api it is exploring will be mounted? I thought the explorer had built-in knowledge of the API it is exploring, but does it fetch it all dynamically via REST when it loads? In which case, the strong-mesh-models module wouldn't need to expose an explorer: pm could instantiate its own if it wanted.